src/Swarm/Util/GameData.hs

```diff
@@ -0,0 +1,119 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+-- Various utilities related to loading game data files.
+module Swarm.Util.GameData where
```

Comment:
@kostmo why do you want to have everything in `Util`?

It is great that you found what part of the util module is used in the Game (and TUI) and even included "game" in the module name.

Please finish the job and move it to Game. :wink:

Reply:
Ok, moved to `Swarm.Game.ResourceLoading`.